lms/djangoapps/instructor/tests/test_receivers.py

```diff
@@ -40,8 +41,6 @@ def setUp(self):
             display_name="Overview"
         )
 
-        self.user = User.objects.create(email='testuser@edx.org', username='testuser', password='testpassword', is_active=True)
```

Comment:
@mattdrayer As these now inherit from ModuleStoreTestCase, it's a redundant declaration. The super makes self.user already.